puppet/manifests/default.pp

```diff
@@ -30,8 +79,27 @@
   }
 }
 
+# This needs to be on exactly two nodes.
+class gluster_cluster {
+  include glusterfs_common
+
+  gluster_peer { ['controller.seed-stack.local', 'worker.seed-stack.local']: }
```

Comment:
We don't actually declare the service anywhere, we just rely on the package setting it up. That means the autorequire won't be applied.

I'll add an autorequire on the package as well so we don't have any problems with this in the future.